docs/pages/versions/unversioned/sdk/tracking-transparency.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Tracking Transparency
+sourceCodeUrl: 'https://github.com/expo/expo/tree/master/packages/expo-tracking-transparency'
+---
+
+import APISection from '~/components/plugins/APISection';
+import InstallSection from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+import SnackInline from '~/components/plugins/SnackInline';
+
+A library for requesting permission to track the user or their device. Examples of data used for tracking include email address, device ID, advertising ID, etc. This permission is only necessary on iOS 14.5 and higher.
```

Comment:
we should be clear that if you call it on iOS < 14.5 then it will always resolve to granted. may want to update the docblocks to indicate this as well

Reply:
turns out this method can also return denied on iOS 14 even before 14.5, updated the docs